aggregator.py

```diff
@@ -605,6 +617,10 @@ def __init__(self, hostname, interval=1.0, expiry_seconds=300, formatter=None, r
             'ms': Histogram,
             's': Set,
         }
+        self.histogram_aggregates = histogram_aggregates\
```

Comment:
That was my first approach, before realizing that no metric specific (say Histograms for instance) code was in this class, I can put that here but I find it lacking consistency a bit. Maybe I should just refactor more things :unamused: